package-lock.json

```diff
@@ -4620,7 +4630,7 @@
       "dependencies": {
         "pify": {
           "version": "2.3.0",
-          "resolved": "https://registry.npmjs.org/pify/-/pify-2.3.0.tgz",
+          "resolved": "http://registry.npmjs.org/pify/-/pify-2.3.0.tgz",
```

Comment:
did we ever figure out what was causing the `https <-> http` diffs?

Reply:
I really do not know, googling it looks like it's still a known issue. I can go back and replace http with https manually though, the links I've read suggested that.